### DISCUSSION TITLE
Calling Fastlane action in a ruby script

### DISCUSSION THREAD

**Discussion by adamtharani** [fastlane/fastlane#18380] [Q&A]
Hello All,

I'm looking for some guidance on if it's possible to invoke the `upload_to_testflight` from an existing ruby script?

Any help is appreciated. :)

**Comment by dinsen** [comment]
Hi @adamtharani 

Try look here:
https://github.com/fastlane/fastlane/issues/13734

**Reply by adamtharani** [reply]
Thanks I will give it a read :)
